app/Helpers/database/player-rank.php

```diff
@@ -8,14 +8,13 @@
 use Illuminate\Support\Carbon;
 use Illuminate\Support\Facades\Cache;
 
-function SetUserUntrackedStatus(User $user, int $isUntracked): void
+function SetUserUntrackedStatus(User $user, bool $isUntracked): void
 {
     $user->Untracked = $isUntracked;
+    $user->unranked_at = $isUntracked ? now() : null;
     $user->save();
 
-    PlayerRankedStatusChanged::dispatch($user, (bool) $isUntracked);
-
-    // TODO update games that are affected by this user's library
```

Comment:
The `PlayerRankedStatusChanged` event triggers `DispatchUpdateGameMetricsForGamesPlayedByUserJob` jobs.